tools/indextemplate/indextemplate_test.go

```diff
@@ -319,7 +319,7 @@ func TestHandleIndexBad(t *testing.T) {
 				t.Fatalf("returned %v and did not raise the expected error", roots)
 			}
 
-			assert.Equal(t, err.Error(), testcase.expected)
+			assert.Equal(t, testcase.expected, err.Error())
```

Comment:
Good catch.  Addressed this and other instances [here][1].

[1]: https://github.com/xiekeyang/oci-discovery/pull/29/commits/3dc6bd8c4ed247c737ae788ca467916706275600